.github/workflows/job-postcommit-java-examples-spark.yml

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+# Runs PostCommit Java Examples Spark via cron and on push to master
+
+name: PostCommit Java Examples Spark
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+permissions: read-all
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          submodules: false
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+
+  postcommit-java-examples-spark:
+    needs: set-properties
+    name: Run PostCommit Java Examples Spark
+    runs-on: [self-hosted, ubuntu-20.04]
+    strategy:
+      fail-fast: false
+      matrix: #JDK17 execution will fail due to new access policies applied to the core JDK libraries starting at JDK 16.
+        java: ${{fromJson(needs.set-properties.outputs.properties).JavaTestProperties.SUPPORTED_VERSIONS}}
+        spark: ${{fromJson(needs.set-properties.outputs.properties).JavaTestProperties.SPARK_VERSIONS}}
+        exclude:
+          - java: "11"
+            spark: "2"
```

Comment:
Is this excluded for the same reason as 17? If not, please comment to explain

Reply:
Sorry to be pedantic, but could you please add some more detail here? Compatibility issues doesn't really tell me much about what the problem is

Reply:
> Sorry to be pedantic, but could you please add some more detail here? Compatibility issues doesn't really tell me much about what the problem is

Oh, no worries, Danny!
About the compatibility issues, Spark v2 dropped support for JDK9+. Spark v3 does work with JDK9+. This thread goes into further detail: https://issues.apache.org/jira/browse/SPARK-24421

Should I add the comment mentioning that?
Thanks!

Reply:
Yep - just something like `Spark v2 dropped support for JDK9+` is great, thanks!